src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java

```diff
@@ -34,7 +34,6 @@
 import graphql.schema.GraphQLObjectType;
 import graphql.schema.GraphQLSchema;
 import graphql.schema.GraphQLType;
```

Comment:
Git is extremely confused by the diff in this file. All other files are pretty much the result of Git cherry pick - this one is the only problematic one

There were major structural changes to this file to prepare for the defer feature to be released in v22. Of course Git can't work out what to do

Instead I've copy pasted the file from master, and deleted any defer references. This is essentially a copy of the non-defer pathway in master.

Reply:
I think this is a good idea.  So basically this is master code without the defer code added.  Seems like a great idea.

The tests have not been changed AND they all pass so great approach